R/pkg/R/generics.R

```diff
@@ -918,6 +918,14 @@ setGeneric("cbrt", function(x) { standardGeneric("cbrt") })
 #' @export
 setGeneric("ceil", function(x) { standardGeneric("ceil") })
 
+#' @rdname collect_list
```

Comment:
these are under `###################### Expression Function Methods ##########################`
which doesn't seem like the right group